sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProperties.cs

```diff
@@ -31,6 +31,11 @@ public class EventHubProperties
         ///
         public string[] PartitionIds { get; }
 
+        /// <summary>
+        /// If the Event Hubs namespace has geo-replication enabled.
+        /// </summary>
```

Comment:
```suggestion
        /// <summary>
        ///   A flag indicating whether or not the Event Hub has geo-replication enabled.
        /// </summary>
        ///
```

This exists on the namespace level, but these properties are scoped to the Event Hub, so we should try to keep that focus.